src/bindings/c/src/common.h

```diff
@@ -35,9 +35,21 @@
     CATCH_OV_EXCEPTION(NETWORK_NOT_READ, NetworkNotRead)      \
     CATCH_OV_EXCEPTION(INFER_CANCELLED, InferCancelled)       \
     catch (...) {                                             \
-        return ov_status_e::UNEXPECTED;                       \
+        return ov_status_e::UNKNOW_EXCEPTION;                 \
     }
 
+#define GET_PROPERTY_FROM_ARGS_LIST                                \
+    std::string property_key = va_arg(args_ptr, char*);            \
+    ov::Any value = {};                                            \
+    if (check_enum_property(property_key)) {                       \
+        int enum_value = va_arg(args_ptr, int);                    \
```

Comment:
I think you can work with enum values as with strings.
I hope names should be fixed on cpp side.

Reply:
Got it, keep all keys as string value, include enum property value, we can define string for each enum value in cpp files. 
One question, do we need expose these string of enum value to user?  just like string of property key.

Reply:
I think in this case we don't need to have enums on C API.

But in case if user wants to work with enum on C level, I think we can have helpers like:
```
enum MyEnum {
};

char* my_enym_to_string(MyEnum);
```

@riverlijunjie What do you think?

Reply:
I think maybe it will be better to abandon enum value if we apply string for all property value. Clear simple interface without  too many helper functions will be more friendly to user.

Reply:
done